### PR TITLE
Fixes/wrangle home

### PR DESCRIPTION
**Pull request by smritiDivami**
# TITLE

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)

### CODE REVIEW

File: app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+export interface IPlugin {
+  plugin: IPluginObject;
+  schema: ISchema;
+}
+
+export interface IPluginObject {
+  name: string;
+  type: string;
+  properties: {};
```
Comment (adityadivami) [inline app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts]: Record<string, unknown>

File: app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  artifact: IArtifact;
+}
+
+export interface ISchema {
+  type: string;
+  name: string;
+  fields: [
+    {
```
Comment (adityadivami) [inline app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts]: separate interface

**Review by adityadivami — CHANGES_REQUESTED** [review]

File: app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts (hunk 1 above)
Comment (smritiDivami) [reply app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts]: Done

File: app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts (hunk 2 above)
Comment (smritiDivami) [reply app/cdap/components/WrangleHome/Components/OngoingDataExploration/types.ts]: Done

**Pull request merged**